keps/prod-readiness/sig-node/4622.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4622
+beta:
+  approver: "@johnbelamaric"
```

Comment:
I can take this one (unless you want it @johnbelamaric)

Reply:
Thanks, I didn't know which approver I should add. let me take this to you.